client/components/site-selector/index.jsx

```diff
@@ -222,8 +224,37 @@ export default React.createClass( {
 
 			const isSelected = this.isSelected( site );
 
-			if ( isSelected && this.props.hideSelected ) {
-				return;
+			return (
+				<Site
+					site={ site }
+					href={ siteHref }
+					key={ 'site-' + site.ID }
+					indicator={ this.props.indicator }
+					onSelect={ this.onSiteSelect.bind( this, site.slug ) }
```

Comment:
ESLint warning here and in a few other places in the file: [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md). From past discussion, seems to be a performance worry about creating new functions for every render. Not really sure what our recommendation is for these cases? Replacing with inline function results in the same warning.

Ref: p4TIVU-2nh-p2